src/components/ResultsListItem.vue

```diff
@@ -0,0 +1,27 @@
+<template>
+  <section class='item-card'>
+    <button @click="$emit('save:park', parkName)" class='save-button'>SAVE</button>
+    <h1>{{result.name}}</h1>
+    <h1>{{result.formatted_address}}</h1>
+  </section>
```

Comment:
Is there any other data that we may potentially want to add for a list item result? Perhaps adding the review rating for a park, and their current distance away from that park, would help with consumer empathy. Not something to worry about this branch, but something that I thought about looking at it from the view of a user. Thoughts?

Reply:
I think including name, distance away (if possible), and rating would be three key pieces of data to show on each list item.  The user will still be able to see more details about the place with the `ResultsItemDetails` component.

With that said, are we thinking that maybe we want to include more information in `ResultsListItem` so that the user is only prompted to view `ResultsItemDetails` if coming from the map list view?

Reply:
My opinion is that the user should have access to `ResultsItemDetails` from either the list view or the map view of results, but if that's the case, the amount shown in list view should be pretty minimal to given our API doesn't provide an extensive amount of data.  Willing to go either way, though, if we decide this isn't the most optimal way.

Reply:
Hmmm, I'm inclined to agree with Nicole that the detail view should be available whether a user is viewing the list or map version of the results. I do think distance from location would be nice on the list view, especially since we added the 'search near me' functionality. On the map view is visible just by looking at the map but it would be nice to have on the list view especially in terms of accessibility for screen readers.

Reply:
Since we pass the full result object into `ResultsListItem` we can refactor to add any data we want, but I agree that the item should be pretty minimal and the details should include the more extensive info. I also agree that the marker and the list item should include the same info. 
Happy we're on the same page here!